src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -46,6 +47,8 @@ public partial class DynamoModel : INotifyPropertyChanged, IDisposable, IEngineC
         public static readonly int MAX_TESSELLATION_DIVISIONS_DEFAULT = 128;
 
         #region private members
+
+        private readonly string geometryFactoryPath;
```

Comment:
I am looking forward to removing `geometryFactoryPath` and `MAX_TESSELLATION_DIVISIONS_DEFAULT` from Dynamo.  :)